drivers/spi/spi_ll_stm32.c

```diff
@@ -726,7 +728,7 @@ static int transceive_dma(const struct device *dev,
 	/* Set buffers info */
 	spi_context_buffers_setup(&data->ctx, tx_bufs, rx_bufs, 1);
 
-#if defined(CONFIG_SOC_SERIES_STM32H7X)
+#if defined(CONFIG_SOC_SERIES_STM32H7X) || defined(CONFIG_SOC_SERIES_STM32U5X)
```

Comment:
Can we use `DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)` here?
`st,stm32h7-spi` compatible was made specifically to handle these cases specific to H7/U5 variations.

Note: This needs to verify of this is compatible with MP1, which use same compatible

Reply:
ok